File: src/my_project_typescript/easy_problems_typescript/from51To100/generateStringCharacterOddCounts.ts
function generateTheString(n: number): string {

    let firstLetter: string = 'a'
    let secondLetter: string = 'b'
    let answer: string = ''
    if (n%2 == 0){
        answer = firstLetter.repeat(n-1) + secondLetter
    }
    else {
        answer = firstLetter.repeat(n)
    }

    return answer 
};


console.log(generateTheString(4))

console.log(generateTheString(2))

console.log(generateTheString(7))

File: src/my_project/easy_problems/from151To200/generate_string_character_odd_counts.py
from typing import List 

class Solution:
    def generateTheString(self, n: int) -> str:
        first_letter: str = "a"
        second_letter: str = 'b'
        answer: str = ''
        if n%2 == 0:
            answer = first_letter*(n-1) + second_letter
        else:
            answer = first_letter*n 
        return answer 


solution = Solution()

print(solution.generateTheString(n= 4))

print(solution.generateTheString(n=2))

print(solution.generateTheString(n=7))
